frontend/src/components/Gym/ConfirmationModal.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import Button from '../Button'
+import styled from 'styled-components'
+
+import 'antd/dist/antd.css'
+
+const OverlayContainer = styled.div<{ overlayBackground: string; isModalOpen: boolean }>`
+  position: fixed;
+  width: 100%;
+  height: 100%;
```

Comment:
would these (height and width) be necessary if the top, left, right and bottom are set?